resources/assets/components/CampaignInbox/index.js

```diff
@@ -81,7 +82,7 @@ class CampaignInbox extends React.Component {
         const newState = {...previousState};
         const user = newState.posts[postId].user;
 
-        newState.posts[postId] = data;
+        newState.posts[postId] = data['data'];
```

Comment:
Should we rename the `data` parameter to `json` or `response` perhaps? `data['data']` feels awkward.